test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -13,6 +13,17 @@
 SMDEBUG_EC2_CPU_INSTANCE_TYPE = get_ec2_instance_type(default="c5.9xlarge", processor="cpu")
 
 
+# TODO: Remove this condition from the gpu smdebug test once the test timeout problems have been resolved
+def _skip_tf1_p28x():
+    """
+    Temporary boolean function to skip tensorflow1 p2.8x tests
+    :return: bool
+    """
+    images = get_dlc_images()
+    return SMDEBUG_EC2_GPU_INSTANCE_TYPE == "p2.8xlarge" and is_tf1(images)
```

Comment:
im testing this in PR 390, i don't think this test gets skipped as expected, maybe due to the fact that is_tf1 isn't work as expected when called on all images. because this needs to be expedited, I am ok with using the previous version of skipping smdebug test that you had. does EKS test still need to be skipped?

Reply:
Based on discussion, This morning, I think the idea is skip for now to complete process quickly, We will merge the PR of EKS scheduling later.